internal/cli/cli.go

```diff
@@ -233,9 +234,63 @@ func documentGet(serverURL string, apiKey string, id string, idIsShareID bool) e
 
 	b, err := json.MarshalIndent(doc, "", "  ")
 	if err != nil {
-		return fmt.Errorf("failed marshalling document with id '%s: %w", id, err)
+		return fmt.Errorf("failed marshalling document with id '%s': %w", id, err)
 	}
 	fmt.Println(string(b))
 
 	return nil
 }
+
+func documentUpdate() *cobra.Command {
+	var title, text string
+	var append, publish bool
+
+	cmd := &cobra.Command{
+		Use:   "update",
+		Short: "Update an existing document",
+		Long:  "Update an existing document's title, text etc. properties",
+		Args:  cobra.MinimumNArgs(1),
```

Comment:
Do we really only have 1 minimum argument? 🤔 
The one argument is the `documentId`, which is obviously required.
But why should we update a document without any updates? 😁 
Should we go with 2 here? 🤷